openhtf/io/user_input.py

```diff
@@ -31,17 +31,20 @@
 import threading
 import uuid
 
-import gflags
+from openhtf.util import argv
+
 
 if platform.system() != 'Windows':
   import termios
 
 _LOG = logging.getLogger(__name__)
 
-FLAGS = gflags.FLAGS
-gflags.DEFINE_integer('prompt_timeout_s',
-                       None,
-                      'User prompt timeout in seconds.')
+ARG_PARSER = argv.ModuleParser()
+ARG_PARSER.add_argument(
+    '--prompt_timeout_s', type=int, action=argv.StoreInModule,
+    target='openhtf.io.user_input.DEFAULT_TIMEOUT_S',
```

Comment:
I went with `'%s.DEFAULT_TIMEOUT_S' % __name__`. `join` seems overkill for this, I'd be okay with switching to `__name__ + '.DEFAULT_TIMEOUT_S'`